spec/views/catalog/bulk_upload_form.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+RSpec.describe 'catalog/bulk_upload_form.html.erb', :type => :view do
+
+  before :each do
+    @obj = Dor.find 'druid:hv992ry2431'
```

Comment:
i'm in favor of more mocking and less actual talking to fedora and solr in unit tests.